scripts/gtk-bundle-from-msys2.sh

```diff
@@ -306,7 +302,7 @@ full download URL as used to create this installation.
 
 sort.exe is extracted from the ZIP archive at
 ${UNX_UTILS_URL}.
-Sort version: ${sort_version}
+Sort version: (textutils) 2.1
```

Comment:
We hardcode the version now because we cannot execute the 32bit `sort` command anymore. I think this is ok as long as we have use the fixed version which we host ourselves (https://download.geany.org/contrib/UnxUpdates.zip).

Ideally, at some point we find a usable 64bit binary for "sort".